build/yamls/antrea-windows-containerd-with-ovs.yml

```diff
@@ -25,11 +45,28 @@ data:
     $mountPath = $env:CONTAINER_SANDBOX_MOUNT_POINT
     $mountPath = ($mountPath.Replace('\', '/')).TrimEnd('/')
     $env:PATH = $env:PATH + ";$mountPath/Windows/System32;$mountPath/openvswitch/usr/bin;$mountPath/openvswitch/usr/sbin"
+    $OVSDriverDir="$mountPath\openvswitch\driver"
+    $WorkDir="$mountPath\openvswitch\"
+
+    # Check if OVSExt driver is already installed
+    $driverStatus = netcfg -q ovsext                                                                           
+    if ($driverStatus -like '*not installed*') { 
+    # Install OVS Driver
+    net start vmms
+    cd $mountPath/openvswitch/driver
```

Comment:
Does it work if you use command `netcfg -l $mountPath/openvswitch/driver/ovsext.inf -c s -i OVSExt`? Then you don't need to switch the paths